example_2/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2_control_demo_example_2</name>
+  <version>0.0.0</version>
+  <description>Demo package of `ros2_control` hardware for DiffBot.</description>
+
+  <maintainer email="denis@stogl.de">Denis Štogl</maintainer>
```

Comment:
you can add yourself too :)

```suggestion
  <maintainer email="denis@stoglrobotics.de">Dr. Denis Štogl</maintainer>
```

Reply:
fine, as maintainer or author? what fits best then? and do i have to add my PhD too? 😋

Reply:
As maintainer, this fits it better for now.

You can add you PhD if you have it :D